DataBase/dataBase.py

```diff
@@ -1,166 +1,131 @@
 import psycopg2
 from configparser import ConfigParser
+from contextlib import contextmanager
 
-def add_user(username, password):
-    cursor, conn = connect()
-
-    query = "insert into users (username, password) values (%s, %s)"
-
-    cursor.execute(query, (username, password))
-    conn.commit()
-
-    conn.close()
-
-def connect():
+@contextmanager
+def get_db_connection():
```

Comment:
establishing DB connection manually using connect function is error-prone, instead opt for a context manager to ensure safely opening and closing connection